### PR TITLE
Update dependencies for Node.js (2017-07-12)

### PR DESCRIPTION
**Pull request by ykzts**
### before

```console
$ yarn outdated
yarn outdated v0.27.5
Package                     Current Wanted Latest Package Type    URL                                                               
@storybook/addon-actions    3.1.6   3.1.8  3.1.8  devDependencies https://github.com/storybooks/storybook/tree/master/addons/actions
@storybook/react            3.1.6   3.1.8  3.1.8  devDependencies https://github.com/storybooks/storybook/tree/master/apps/react    
autoprefixer                7.1.1   7.1.2  7.1.2  dependencies    https://github.com/postcss/autoprefixer#readme                    
babel-loader                7.1.0   7.1.1  7.1.1  dependencies    https://github.com/babel/babel-loader                             
babel-preset-env            1.5.2   1.6.0  1.6.0  dependencies    https://babeljs.io/                                               
chai                        4.0.2   4.1.0  4.1.0  devDependencies http://chaijs.com                                                 
emojione                    2.2.7   2.2.7  3.1.0  dependencies    http://www.emojione.com                                           
eslint                      3.19.0  3.19.0 4.2.0  devDependencies http://eslint.org                                                 
eslint-plugin-jsx-a11y      4.0.0   4.0.0  6.0.2  devDependencies https://github.com/evcohen/eslint-plugin-jsx-a11y#readme          
eslint-plugin-react         6.10.3  6.10.3 7.1.0  devDependencies https://github.com/yannickcr/eslint-plugin-react                  
extract-text-webpack-plugin 2.1.2   2.1.2  3.0.0  dependencies    http://github.com/webpack-contrib/extract-text-webpack-plugin     
intl-relativeformat         1.3.0   1.3.0  2.0.0  dependencies    https://github.com/yahoo/intl-relativeformat                      
js-yaml                     3.8.4   3.9.0  3.9.0  dependencies    https://github.com/nodeca/js-yaml                                 
jsdom                       11.0.0  11.1.0 11.1.0 devDependencies https://github.com/tmpvar/jsdom#readme                            
postcss-smart-import        0.7.4   0.7.5  0.7.5  dependencies    https://github.com/sebastian-software/postcss-smart-import#readme 
react-router-scroll         0.4.1   exotic exotic dependencies    ytase/react-router-scroll#build                                   
sinon                       2.3.5   2.3.7  2.3.7  devDependencies http://sinonjs.org/                                               
substring-trie              1.0.0   1.0.1  1.0.1  dependencies                                                                      
webpack                     3.0.0   3.2.0  3.2.0  dependencies    https://github.com/webpack/webpack                                
webpack-manifest-plugin     1.1.0   1.1.2  1.1.2  dependencies    https://github.com/danethurber/webpack-manifest-plugin            
Done in 2.11s.
```

### after

```console
$ yarn outdated
yarn outdated v0.27.5
Package                Current Wanted Latest Package Type    URL                                                     
emojione               2.2.7   2.2.7  3.1.0  dependencies    http://www.emojione.com                                 
eslint                 3.19.0  3.19.0 4.2.0  devDependencies http://eslint.org                                       
eslint-plugin-jsx-a11y 4.0.0   4.0.0  6.0.2  devDependencies https://github.com/evcohen/eslint-plugin-jsx-a11y#readme
eslint-plugin-react    6.10.3  6.10.3 7.1.0  devDependencies https://github.com/yannickcr/eslint-plugin-react        
react-router-scroll    0.4.1   exotic exotic dependencies    ytase/react-router-scroll#build                         
Done in 2.02s.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,17 +49,17 @@
...
+    "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
     "font-awesome": "^4.7.0",
     "glob": "^7.1.1",
     "http-link-header": "^0.8.0",
     "immutable": "^3.8.1",
     "intersection-observer": "^0.3.2",
     "intl": "^1.2.5",
-    "intl-relativeformat": "^1.3.0",
+    "intl-relativeformat": "^2.0.0",
```
Comment (ykzts) [inline package.json]: - [Release v2.0.0 — Upgraded to Latest intl-messageformat · yahoo/intl-relativeformat](https://github.com/yahoo/intl-relativeformat/releases/tag/v2.0.0)

> - Upgrade intl-messageformat to ^2.0.0 ([#50](https://github.com/yahoo/intl-relativeformat/pull/50), **@ jasonmit**)

Comment (ykzts) [inline package.json]: [intl-messageformat](https://github.com/yahoo/intl-messageformat)

- [Release v2.0.0 — Return zero · yahoo/intl-messageformat](https://github.com/yahoo/intl-messageformat/releases/tag/v2.0.0)

> - zero values should not return an empty string ([#162](https://github.com/yahoo/intl-messageformat/pull/162), **@ cwtuan**, **@ rlivsey**, **@ jasonmit**)

File: package.json [hunk 2]
```diff
@@ -49,17 +49,17 @@
     "es6-symbol": "^3.1.1",
     "escape-html": "^1.0.3",
     "express": "^4.15.2",
-    "extract-text-webpack-plugin": "^2.1.2",
+    "extract-text-webpack-plugin": "^3.0.0",
```
Comment (ykzts) [inline package.json]: - [Release Release v3.0.0 · webpack-contrib/extract-text-webpack-plugin](https://github.com/webpack-contrib/extract-text-webpack-plugin/releases/tag/v3.0.0)

Support for webpack v2.x has been dropped.

**Review by Gargron — APPROVED** [review]

**Comment by sorin-davidoi** [comment]
Should we look into automating this? A quick search lead to [dependabot](https://dependabot.com/):
 - Supports Ruby and JS
 - Can be configured to create a PR on a weekly basis
 - Links to changelogs
 - **Shows which version bumps created problems for other people**

**Pull request merged**